docker-compose.yml

```diff
@@ -52,11 +58,14 @@ services:
     healthcheck:
       test: curl --fail http://localhost:3000/meta || exit 1
       interval: 20s
-      retries: 10
+      retries: 15
       start_period: 20s
       timeout: 10s
     depends_on:
-      - 'postgres'
+      postgres:
+        condition: service_started
+      chain:
+        condition: service_healthy
```

Comment:
It was being a slow startup for me because it would call out to the chain right away and end up with an empty genesis hash. It would still try to build and only then fail / restart.

With this check it will have the hash and only do the slow step once now.